common/services/prismic/exhibitions.js

```diff
@@ -209,6 +212,43 @@ export async function getExhibitionsPast(req: Request) {
   return paginatedResults;
 }
 
+export async function getExhibitionsCurrentAndComingUp(req: Request) {
+  const order = 'asc';
+  const predicates = [
+    Prismic.Predicates.dateAfter(endField, london().toDate())
+  ];
+  const paginatedResults = await getExhibitions(req, {
+    predicates, order
+  });
+
+  // We order the list this way as, from a user's perspective, seeing the
+  // temporary exhibitions is more urgent, so they're at the front of the list,
+  // but there's no good way to express that ordering through Prismic's ordering
+  const groupedResults = paginatedResults.results.reduce((acc, result) => {
+    // Wishing there was `groupBy`.
```

Comment:
https://www.npmjs.com/package/lodash.groupby ?

Reply:
I'm trying to be conservative as to where we need things as we'll start packaging these on the client too.

Reply:
Seems to be in the opening times.

Nice to try keep it slim though as if that show up to be a large part of the bundle, we can remove it easier if there is only 1 use...

Basically it would be nice to move to monitoring the bundle size, but first, we need the apps running.